shortcut/base.py

```diff
@@ -15,77 +17,162 @@ class ShortCutter(object):
         s.create_menu_shortcut("python")
     """
 
-    def __init__(self):
+    def __init__(self, err_file=None):
+        self.err_file = sys.stderr if (err_file is None) else err_file
```

Comment:
Whats going on here? 

I can understand the requirement to write errors to a file but if someone if using shortcuts API they would want errors raised not written to stderr where they difficult to capture.

If someone had a requirement to log errors to file I would expect them to do it outside of the API and in their own program.

If the requirement is to write to an error file using the `shortcut` command I would expect this to be implemented in `main`.

Reply:
> If someone had a requirement to log errors to file I would expect them to do it outside of the API and in their own program.

I'll add this as a silent mode as I expect logging errors to a file to be the most common use-case: less code to write. Default behavior still would be raising errors as they are.